### PR TITLE
general fixes to yield-generator.py

### PR DESCRIPTION
**Pull request by urban48**
fixed pep8
better imports
new style class inheritance fixed
made code more readble in general

### CODE REVIEW

**Comment by adlai** [comment]
I'm in favor of this kind of cleanup, but this commit also *adds* extra whitespace...

**Comment by urban48** [comment]
@adlai well, its a change from tab indentation to space indentation, because of pep8

File: yield-generator.py [hunk 1]
```diff
@@ -1,158 +1,160 @@
...
+from lib.maker import Maker
+from lib.irc import IRCMessageChannel, random_nick
 
 txfee = 1000
-cjfee = '0.002' # 0.2% fee
+cjfee = '0.002'  # 0.2% fee
 nickname = random_nick()
 nickserv_password = ''
-minsize = int(1.2 * txfee / float(cjfee)) #minimum size is such that you always net profit at least 20% of the miner fee
+minsize = int(
```
Comment (adlai) [inline yield-generator.py]: this?

**Comment by adlai** [comment]
Perhaps it's better to pool your efforts with @ghtdak on #254

File: yield-generator.py (hunk 1 above)
Comment (urban48) [inline yield-generator.py]: fixed :)

**Comment by urban48** [comment]
i will, tnx

**Comment by AdamISZ** [comment]
Formatting has been addressed for the whole codebase by #345 and the is in the develop branch
#349 has created alternative yieldgenerator scripts.

If there's something still worth changing, please submit a new PR against the develop branch, but bear in small refactoring there will be low priority for now. See #343 for the current plan for progress.

**Pull request closed without merging**